yandexcloud/_channels.py

```diff
@@ -30,35 +40,69 @@ def __init__(self, client_user_agent=None, **kwargs):
             endpoint=self._endpoint,
         )
 
-        self._unauthenticated_channel = None
-        self._channels = None
         self._client_user_agent = client_user_agent
-
-    def channel_options(self):
-        return tuple(
+        self._config_endpoints = copy(endpoints)
```

Comment:
It is looks like it could be None, but later you're using `service not in self._config_endpoints`.
Maybe `self._config_endpoints = {} if endpoints is None else dict(endpoints)` would be better?

Reply:
done